Gemfile

```diff
@@ -128,11 +128,12 @@ group :test do
   gem "webmock", require: false
   gem "fakeweb", require: false
   gem "simplecov", require: false
-  gem "selenium-webdriver", "~> 4.14", require: false
-  gem "selenium-devtools", require: false
   gem "test-prof"
   gem "rails-dom-testing", require: false
   gem "minio_runner", require: false
+  gem "selenium-webdriver", "~> 4.14", require: false
+  gem "selenium-devtools", require: false
```

Comment:
```suggestion
```

I think these lines can be removed.

Reply:
I will check later but I think selenium is still used in some migration too. But they could potentially load it differently I guess.

Reply:
Hmm I double checked and yes I think google ones is loaded inline so we can probably remove it safely, and it's in test group anyways.